api/handler/copy.go

```diff
@@ -101,6 +101,15 @@ func (h *handler) CopyObjectHandler(w http.ResponseWriter, r *http.Request) {
 			h.logAndSendError(w, "access control list not supported", reqInfo, s3errors.GetAPIError(s3errors.ErrAccessControlListNotSupported))
 			return
 		}
+		if state == data.BucketACLBucketOwnerPreferred {
```

Comment:
I expect ACLs to work properly in this case.

Reply:
Yeah, it seems it should work as was before. Removed